packages/ses/scripts/bundle.js

```diff
@@ -15,40 +17,59 @@ const write = async (target, content) => {
   await fs.promises.writeFile(location, content);
 };
 
-const main = async () => {
+/**
+ * @param {object} [options]
+ * @param {string} [options.buildType] Suffix used to build special bundles (e.g. 'hermes')
+ */
+const writeBundle = async ({ buildType } = {}) => {
   const text = await fs.promises.readFile(
     fileURLToPath(`${root}/package.json`),
     'utf8',
   );
   const packageJson = JSON.parse(text);
   const version = packageJson.version;
 
+  const moduleTransforms = {};
+
+  let bundleFilePaths = [
+    'dist/ses.cjs',
+    'dist/ses.mjs',
+    'dist/ses.umd.js',
+    'dist/lockdown.cjs',
+    'dist/lockdown.mjs',
+    'dist/lockdown.umd.js',
+  ];
+  let terseFilePaths = ['dist/ses.umd.min.js', 'dist/lockdown.umd.min.js'];
+
+  if (buildType === 'hermes') {
+    bundleFilePaths = ['dist/ses-hermes.cjs'];
+    terseFilePaths = [];
+    Object.assign(moduleTransforms, hermesTransforms);
+  }
+
   const bundle = await makeBundle(
     read,
     pathToFileURL(resolve('../index.js', import.meta.url)).toString(),
+    // @ts-expect-error TS2322 Type '{}' is not assignable to type 'ModuleTransforms'. Index signature for type 'string' is missing in type '{}'.
```

Comment:
Could this be resolved by declaring `{} /* as Record<string, unknown} */` or even `{} /* as ModuleTransform`?

Reply:
running `yarn docs`

<img width="905" alt="Screenshot 2024-09-09 at 2 22 44 PM" src="https://github.com/user-attachments/assets/eb5e76f0-c577-4edb-832b-9a6e21d7a91b">

but `/** @type ModuleTransforms */` is fine ^ after adding the import

`/** @import {ModuleTransforms} from '../../compartment-mapper/types.js' */`

https://github.com/endojs/endo/pull/2334/commits/58bf929eb01c887da0adceb043454919451074dd